### PR TITLE
a start for CI work on 18Feb:

### PR DESCRIPTION
**Pull request by opotowsky**
added: 
   CYM directory with build.sh
   cymetric.fast.run-spec
   (do we also need CYM/run_test.sh?)

### CODE REVIEW

**Comment by scopatz** [comment]
> (do we also need CYM/run_test.sh?)

We might, but that can be added later.

File: CYM/build.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  line=$(grep -i warning doc.out | wc -l)
+  if [ $line -ne 0 ]; then
+    exit 1
+  fi
+  ls -l
+  mv doc "${origdir}/cymetricdoc"
+  cd "$origdir"
+fi
+
+# Regression Testing
```
Comment (scopatz) [inline CYM/build.sh]: I don't think that we need anything under this line.

File: cymetric.fast.run-spec [hunk 2]
```diff
@@ -0,0 +1,8 @@
+run_type    = test
+inputs      = fetch/conda.linux.url,fetch/conda.mac.url, fetch/git.git, fetch/myCiclus.scp,fetch/conda-recipe.git
+remote_pre_declare = CYM/build.sh
+remote_task = CYM/run_test.sh
+platforms   = x86_64_Ubuntu12,x86_64_MacOSX8
```
Comment (scopatz) [inline cymetric.fast.run-spec]: Let's add Ubuntu14 and MacOSX9

File: cymetric.fast.run-spec [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+project     = <a href="http://cyclus.github.io/">Cyclus</a>
```
Comment (scopatz) [inline cymetric.fast.run-spec]: May  want to change the project name here.

File: CYM/build.sh [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+fi
+
+# force cyclus to build with local cymetric
+vers=$(cat cymetric/meta.yaml | grep version)
+read -a versArray <<< $vers
+
+anaconda/bin/conda install --use-local cymetric=${versArray[1]}
+tar -czf results.tar.gz anaconda
+
+cp -r "${WORKDIR}/tests" cymtest
```
Comment (scopatz) [inline CYM/build.sh]: Actually, I don't think we need anything from this line and below.

**Comment by scopatz** [comment]
Thanks for taking a stab at this Arrielle! Do you know how to test if it works?

**Comment by opotowsky** [comment]
I do not! I assumed if I couldn't get one build to work I wouldn't be able to test this. 

(I'll definitely get the changes into the PR once I'm back on a computer!)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**